src/android/PerfChart.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { PerfherderGraphContainer } from '../utils/PerfherderGraphContainer';
+import { COMBOS, PLATFORMS } from './config';
+import { selectFrom } from '../vendor/vectors';
+import { TimeDomain } from '../vendor/jx/domains';
+
+const timeDomain = new TimeDomain({ past: '3month', interval: 'day' });
+
+const PerfChart = ({ title, suite }) => PLATFORMS.map(({ label, filter, id }) => (
```

Comment:
@ajinkabeer I do not think this should be a separate function:  It is only used in one place.  Putting the component in its own should also not be done: It is not useful anywhere else.  But, you may leave it.

Please come up with a better name: Since everything is "perf"-something, this current name is not descriptive.

Reply:
@klahnakoski Yes. Now I think It was unnecessary. Better to switch back to the previous code in `index.jsx` since it's not used elsewhere.